public/js/components.js

```diff
@@ -1,122 +0,0 @@
-// The Grid component allows an element to be located
```

Comment:
so it looks like you're working on a side scroll game component. not quite sure if we're at a place where this would be a good idea. talk with pablo. and maybe you should work on this is a separate branch until it fully works.